DocAI Incubator Tools/best-practices/Child_Entity_Tag_Using_Header_Keyword.ipynb

```diff
@@ -0,0 +1,726 @@
+{
```

Comment:
<p>Most of the functions in this code block are incredibly long and difficult to read. Could they be split up into multiple functions to simplify readability?</p><p> </p><p>When working with the Document object, it's best to use the object type rather than dicts.</p><p> </p><p>There's also no type annotations for the functions - which should be added.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/document-ai-samples/pull/640/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>